modules/squish/config.py

```diff
@@ -1,5 +1,5 @@
 def can_build(env, platform):
-    return True
+    return False
```

Comment:
Does this mean etcpak supersedes squish and we can drop it too? If so, we should drop it.

Reply:
Squish provides DXT1, DXT3, DXT5, BC4 and BC5.

Since all we use is dxt3, dxt5 and etc1, etc2 I disabled it to see if it mattered.

For BC6 we can use Betsy but I still couldn't find a good BC7 compressor.

Reply:
I'm using etcpak to encode and squish to decode because the changes required more engineering to expose etcpak's decoder.